apps/api/src/app/app.controller.ts

```diff
@@ -20,4 +27,9 @@ export class AppController {
     getWelcomeMessage(): Message {
         return { message: 'Welcome to the COSCRAD API!' };
     }
+
+    @Get('avcoms')
```

Comment:
Just wondering about `avcoms`, sometimes 'coms' refers to communications.  Maybe spell it out?

Reply:
Sorry for the confusion- this endpoint will be deleted later. But I can definitely spell it out for now.

Reply:
done